KsApi/queries/templates/FetchBackerProjectsQueryRequestForTests.graphql

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
This is only added to the test target - it's just here so that the next person who needs to generate one of these queries (which will probably be me!) doesn't have to figure out the full query text again 😁